libs/island-ui/core/src/lib/CategoryCard/CategoryCard.tsx

```diff
@@ -36,6 +37,7 @@ export type CategoryCardProps = {
   tags?: Tag[]
   tagOptions?: Pick<TagProps, 'hyphenate' | 'truncate' | 'textLeft'>
   href?: string
+  nestedHref?: boolean
```

Comment:
What is a nested href? Is that something we've got in figma as well?

Reply:
Couldn't we rather detect if there is a nested href instead of explicitly telling the component?

Reply:
Also your solution only makes the title clickable instead of the traditional "entire card can be clicked"

Reply:
We could always check if there is a tag passed to the component that has `href` prop. But if we want to have the whole box clickable we need to use router or some other api to allow routing instead of the anchor tag. But how the component is programmed right now allows for nested anchor tags.

Reply:
Just read some things, you are right that <a> inside an <a> is invalid html. 

Perhaps the tags themselves should have an onClick or the card itself. However this issue is in more places, for example the "Lífsviðburðir" section on https://island.is/ also has nested <a> tags.